src/components/CalendarModal/CalendarModal.module.scss

```diff
@@ -6,22 +6,66 @@
   height: 100%;
   background: rgba(0, 0, 0, 0.6);
   text-align: center;
-  z-index: 1;
+  z-index: 10;
```

Comment:
Do we still need this change if the footer is at z-index: -1 now?

Reply:
It is unnecessary, you're right! Changed it back :)